test/framework/k2s/addons.go

```diff
@@ -157,3 +158,28 @@ func GetImagesForAddon(addon Addon) ([]string, error) {
 
 	return lo.Union(images), nil
 }
+
+type AddonsInfo struct {
+}
+
+func NewAddonsInfo() *AddonsInfo {
```

Comment:
Since `AddonsInfo` does provide neither functionality nor state, consider removing it to simplify the usage and code in `suite.go`.